public_html/app/models/User.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+    class User {
+        private $db;
+
+        // Instantiates 'Database' class
+        public function __construct() {
+            $this->db = new Database;
+        }
+
+        // Login User
+        public function login($email, $password) {
```

Comment:
This is model. Not controller. Model is responsible to database not to business logic. Move it to controller.